### ISSUE TITLE
OSError:Model name '/data/ConSERT-master/chinese-roberta-wwm-ext-large' was not found in tokenizers model name list(roberta-base, roberta-large, robert-large-mnli, distilroberta-base, roberta-base-openai-detector, roberta-large-openai-detector)

### ISSUE THREAD

**Issue by HL0718** [yym6472/ConSERT#18]
我按照README文件的知识， 然后将chinese-roberta-wwm-ext-large 预训练模型文件下载好了放在./chinese-roberta-wwm-ext-large 目录下，然后我运行了python3 main.py --no_pair --seed 1 --use_apex_amp --apex_amp_opt_level O1 --batch_size 32 --max_seq_length 40 --evaluation_steps 20 --add_cl --cl_loss_only --cl_rate 0.15 --temperature 0.1 --learning_rate 0.0000005 --chinese_dataset atec_ccks --num_epochs 10 --da_final_1 feature_cutoff --da_final_2 shuffle --cutoff_rate_final_1 0.2 --model_name_or_path ./chinese-roberta-wwm-ext-large --model_save_path ./output/unsup-consert-large-atec_ccks --force_del --patience 10 命令，然后就提示了以上错误，请问这个是什么问题？能帮忙解答一下吗？
